internal/cmd/base/base.go

```diff
@@ -273,6 +275,22 @@ func (c *Command) Client(opt ...Option) (*api.Client, error) {
 
 		c.client.SetRecoveryKmsWrapper(wrapper)
 
+	case c.FlagToken != "":
+		token, err := parseutil.MustParsePath(c.FlagToken)
+		switch {
+		case err == nil:
+		case err.Error() == parseutil.ErrNotParsed.Error():
```

Comment:
Does this really need to be a string comparison? `err == parseutil.ErrNotParse` or some of the `errors.Match` helpers?

Reply:
Changed